### ISSUE TITLE
[Bug Report] When using el-select remote search in el-dialog, the search results are blocked by dialog

### ISSUE THREAD

**Issue by 1008611** [ElemeFE/element#9424]
### Element UI version
1.4.13

### OS/Browsers version
Chrome

### Vue version
2.5.13

### Reproduction Link
[https://jsfiddle.net/C__C/7zvydcgo/](https://jsfiddle.net/C__C/7zvydcgo/)

### Steps to reproduce
1.打开dialog，然后点model之外，关闭dialog，
2.再次打来dialog进行搜索时候，搜索结果被dialog遮挡。

### What is Expected?
搜索结果应该出现在输入框下面

### What is actually happening?
搜索结果被dialog遮挡

**Comment by Hi-Linlin** [comment]
Duplicate of #9315
